samples/Exporters/Console/TestZPagesExporter.cs

```diff
@@ -0,0 +1,69 @@
+﻿// <copyright file="TestZPagesExporter.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+
+using OpenTelemetry.Exporter.ZPages;
+using OpenTelemetry.Exporter.ZPages.Implementation;
+using OpenTelemetry.Trace.Configuration;
+
+namespace Samples
+{
+    internal class TestZPagesExporter
+    {
+        internal static object Run()
+        {
+            var zpagesOptions = new ZPagesExporterOptions() { Url = "http://localhost:7284/rpcz/" };
+            var zpagesExporter = new ZPagesExporter(zpagesOptions);
+            var zpagesProcessor = new ZPagesProcessor(zpagesExporter);
+            ZPagesActivities.RetentionTime = 3600000;
+            var httpServer = new ZPagesExporterStatsHttpServer(zpagesExporter, zpagesProcessor);
+
+            // Start the server
+            httpServer.Start();
+
+            using var openTelemetry = OpenTelemetrySdk.EnableOpenTelemetry(
+                builder => builder
+                    .AddActivitySource("zpages-test")
+                    .UseZPagesExporter(
+                        processorConfigure: processor => processor
+                            .AddProcessor((next) => zpagesProcessor)));
+
+            ActivitySource activitySource = new ActivitySource("zpages-test");
+
+            while (true)
```

Comment:
this run forever?

Reply:
At least, in the example we had the same: https://github.com/open-telemetry/opentelemetry-dotnet/blob/4eece94ec6712da7d7583e9bdc0b965ca90ac106/samples/Exporters/Console/TestZPages.cs#L47-L51

Reply:
Okay got it.